requirements-dev.txt

```diff
@@ -1,6 +1,9 @@
 # Install package into itself.
 -e .
 
+# Install main requirements.
+-r requirements.txt
```

Comment:
After rebasing, the following line will need to be removed:

https://github.com/wagtail/sphinx-wagtail-theme/blob/0dad2f03c5558fd44aacfb8629d4066008af4335/Makefile#L59